initrd-progs/0initrd/init

```diff
@@ -76,13 +76,15 @@ export KERNELVER="$(uname -r)"
 export DISTRODESC="${DISTRO_NAME} ${DISTRO_VERSION} - Linux ${KERNELVER} - `uname -m`"
 
 #precaution - if DISTRO_SPECS was not processed by 3builddistro...
+[ ! "$DISTRO_BDRVSFS" ] && DISTRO_BDRVSFS="bdrv_${DISTRO_FILE_PREFIX}_${DISTRO_VERSION}.sfs"
```

Comment:
@mavrothal Did I mess up initrd in any way? This code is messy and I want to make sure I didn't break anything.